packages/core/admin/admin/src/shared/hooks/useInjectionZone.ts

```diff
@@ -0,0 +1,14 @@
+import { useAdmin } from '../../contexts/admin';
+
+export type InjectionZoneArea = string;
```

Comment:
would it be better to do `${string}.${string}.${string}` to give better typesafety?

Reply:
I think what would really improve this type is to hard-code the fixed list of injection zones we have. I'll have a look later and see if I can extract it. 👍🏼